locale/de.json

```
@@ -276,9 +277,10 @@
 	"pages.content.index.search.placeholder": "Lernstore durchsuchen",
 	"pages.content.index.search_resources": "Ressourcen",
 	"pages.content.index.search_results": "Suchergebnisse für",
-	"pages.content.notification.successMsg": "Material wurde erfolgreich hinzugefügt",
-	"pages.content.notification.errorMsg": "Irgendetwas ist schiefgegangen. Material konnte nicht hinzugefügt werden",
+	"pages.content.notification.errorMsg": "Material konnte nicht hinzugefügt werden",
```

Do we use there short sentence like word lists or should we use complete sentences? I would use the following, while Material should be Lern-Material too? Maybe discuss this with Caro.
```suggestion
	"pages.content.notification.errorMsg": "Das Material konnte nicht hinzugefügt werden",
```